### DISCUSSION TITLE
cursor teleporting to corner & stutters

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18257] [Q&A] [answered]
I've been having this issue for awhile now, consistantly, and It might be an arch linux issue, but even if it is i can't find anything on it...

sometimes while playing (seemingly at random) my cursor will teleport to the bottom right corner, and i will have framerate drops and stutters of input at the same time, sometimes ruining combo.

https://user-images.githubusercontent.com/78441708/168410139-fec6d8f1-48ba-4ba3-95f1-561fb31257ee.mp4

if you look closely you can see the cursor teleport, but the framerate stutters don't show up in the recording. 

[logs.zip](https://github.com/ppy/osu/files/8692155/logs.zip)

i'd say this happens about once every minute. it doesn't seem to happen upon anything.

**Comment by smoogipoo** [comment, marked as answer]
Do you have high precision mode turned on in the options? If so, try turning it off.

**Reply by Axilotl17** [reply]
I turned it off, and for the past 2 weeks had have similar stutters but my cursor doesnt teleport anymore. thanks!
